libflux/Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["flux-core", "flux"]
+resolver = "2"
```

Comment:
apparently workspaces w/ edition 2021 crates default to the v2 resolver, this just makes it explicit and silences a cargo warning